docs/learn/learn-accounts.md

```diff
@@ -390,129 +391,92 @@ Learn more about multisig accounts from our
-sorting, then be aware that the public keys are compared byte-for-byte and sorted ascending before
-being inserted in the payload that is hashed.
-
-:::
+### Making Transactions with a Multi-signature Account
 
-Addresses are deterministically generated from the signers and threshold of the multisig wallet. For
-a code example (in TypeScript) of generating you can view the internals of `@w3f/msig-util`
-[here](https://github.com/lsaether/msig-util/blob/master/src/actions/deriveAddress.ts).
-
-The `@w3f/msig-util` is a small CLI tool that can determine the multi-signature address based on
-your inputs.
-
-```zsh
-$ npx @w3f/msig-util@1.0.7 derive --addresses 15o5762QE4UPrUaYcM83HERK7Wzbmgcsxa93NJjkHGH1unvr,1TMxLj56NtRg3scE7rRo8H9GZJMFXdsJk1GyxCuTRAxTTzU --threshold 1
-npx: installed 79 in 7.764s
---------------------------------
-Addresses: 15o5762QE4UPrUaYcM83HERK7Wzbmgcsxa93NJjkHGH1unvr 1TMxLj56NtRg3scE7rRo8H9GZJMFXdsJk1GyxCuTRAxTTzU
-Threshold: 1
-Multisig Address (SS58: 0): 15FKUKXC6kwaXxJ1tXNywmFy4ZY6FoDFCnU3fMbibFdeqwGw
---------------------------------
-```
+There are three types of actions you can take with a multisig account:
 
-The Polkadot-JS Apps UI also supports multi-sig accounts, as documented in the
-[Account Generation page](learn-account-generation.md#multi-signature-accounts). This is easier than
-generating them manually.
+- Executing a call `asMulti`. This is used to begin or end a multisig transaction.
+- Approving a call `approveAsMulti`. This is used to approve an extrinsic and pass-on to the next signatory (see [example below](#example-using-multi-signature-accounts) for more information).
+- Cancelling a call `cancelAsMulti`.
 
-### Making Transactions with a Multi-signature Account
+:::info
 
-There are three types of actions you can take with a multi-sig account:
+Check out [this page](https://polkadot.js.org/docs/substrate/extrinsics#multisig) for more information about the actions you can take with a multi-signature account.
 
-- Executing a call `as_multi`. 
-- Approving a call `approve_as_multi`.
-- Cancelling a call `cancel_as_multi`.
+:::
 
 In scenarios where only a single approval is needed, a convenience method `as_multi_threshold_1`
-should be used. This function takes only the other signatories and the raw call as its arguments.
+should be used. This function takes only the other signatories and the raw call as its arguments. Note that the Polkadot-JS UI does not have integration for this call because it is not possible to create multisig accounts with `threshold=1`. If you would like to create a multisig with threshold 1 then you can use [txwrapper-core](https://github.com/paritytech/txwrapper-core) which is a tool that Parity supports with new features and weekly updates. There is a detailed [multisig example](https://github.com/paritytech/txwrapper-core/tree/main/packages/txwrapper-examples/multisig) that you can try out and change to see how it works.
```

Comment:
```suggestion
should be used. This function takes only the other signatories and the raw call as its arguments. Note that the Polkadot-JS UI does not have integration for this call because it is not possible to create multisig accounts with `threshold=1`. If you want to create a multisig with threshold 1, then you can use [txwrapper-core](https://github.com/paritytech/txwrapper-core), which is a tool that Parity supports with new features and weekly updates. There is a detailed [multisig example](https://github.com/paritytech/txwrapper-core/tree/main/packages/txwrapper-examples/multisig) that you can try out and change to see how it works.
```